sherpa/astro/ui/tests/test_astro_session.py

```diff
@@ -0,0 +1,26 @@
+#
+#  Copyright (C) 2016  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+from sherpa.astro.ui.utils import Session
+
+
+# bug #303
+def test_show_bkg_model():
```

Comment:
It looks like it doesn't make much of a difference when the bug is fixed, but I also added `show_bkg_source()` and `show_bkg_source('xx')`.